### ISSUE TITLE
A lot of errors with recent update 0.3.2

### ISSUE THREAD

**Issue by firestrife23** [danimtb/dasshio#76]
just updated to 0.3.2 and getting alot of errors:

```
2019-04-14 14:32:56,737 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:32:56,787 | INFO | Finishing sniffing
2019-04-14 14:32:56,787 | INFO | Starting sniffing...
2019-04-14 14:32:57,181 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:32:57,237 | INFO | Finishing sniffing
2019-04-14 14:32:57,237 | INFO | Starting sniffing...
2019-04-14 14:32:57,725 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:32:57,777 | INFO | Finishing sniffing
2019-04-14 14:32:57,777 | INFO | Starting sniffing...
2019-04-14 14:33:02,679 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:33:02,717 | INFO | Finishing sniffing
2019-04-14 14:33:02,718 | INFO | Starting sniffing...
2019-04-14 14:33:03,493 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:33:03,527 | INFO | Finishing sniffing
2019-04-14 14:33:03,527 | INFO | Starting sniffing...
2019-04-14 14:33:04,023 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:33:04,097 | INFO | Finishing sniffing
2019-04-14 14:33:04,097 | INFO | Starting sniffing...
2019-04-14 14:33:04,494 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:33:04,527 | INFO | Finishing sniffing
2019-04-14 14:33:04,528 | INFO | Starting sniffing...
2019-04-14 14:33:05,100 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:33:05,147 | INFO | Finishing sniffing
2019-04-14 14:33:05,147 | INFO | Starting sniffing...
2019-04-14 14:33:05,679 | ERROR | Caught exception in sniff: name 'datetime' is not defined
Traceback (most recent call last):
  File "/dasshio.py", line 170, in <module>
    count=0)
  File "/usr/lib/python3.6/site-packages/scapy/sendrecv.py", line 889, in sniff
    if stop_filter and stop_filter(p):
  File "/dasshio.py", line 30, in arp_display
    current_time = datetime.utcnow()
NameError: name 'datetime' is not defined
2019-04-14 14:33:05,717 | INFO | Finishing sniffing
2019-04-14 14:33:05,717 | INFO | Starting sniffing...
```

**Comment by matthewharrington** [comment]
Same issue here. Cheers, Matt

**Comment by dimmanramone** [comment]
Same here. Exactly the same errors.

**Comment by danimtb** [comment]
Seems the import is missing. Will be fixed soon in 0.3.3
